src/SimpleBusyLoop/SBLmain.cpp

```diff
@@ -0,0 +1,23 @@
+#include "fpcsc/perf_util/sleep.h"
+#include "fpcsc/perf_util/feature_cmd.h"
+
+#include <string>
+
+volatile long Counter;
+
+int main(int argc, char *argv[])
+{
+  long NumIterations = fpcsc::getFeatureValue(argc, argv, "--iterations");
+  long CountTo = fpcsc::getFeatureValue(argc, argv, "--count_to");
+
+  if (!NumIterations || !CountTo)
+    std::__throw_runtime_error("Required feature missing.");
```

Comment:
This is not a public function that should be used here. Just print the message on std::cerr and return an error code.